core/lib/presentation/utils/app_toast.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:core/presentation/extensions/color_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+
+class AppToast {
+  void showToast(String message) {
+    Fluttertoast.showToast(
```

Comment:
why not use toast from GetX?

Reply:
getx not support toast. If you know please send me.

Reply:
:) okie